osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -413,6 +416,24 @@ private void deleteCommentRequest()
             api.Queue(request);
         }
 
+        public void ReportComment(CommentReportReason reason, string comment)
+        {
+            actionsContainer.Hide();
```

Comment:
Not sure reporting a comment should hide all actions? One may want to take a permalink of the comment after reporting it, for example.